trade_client.py

```diff
@@ -34,7 +34,7 @@ def place_order(base,quote, amount, side, last_price):
     'DOT', 'USDT', 50, 'buy', 400
     """
     try:
-        order = Order(amount=str(float(amount)/float(last_price)), price=last_price, side=side, currency_pair=f'{base}_{quote}')
+        order = Order(amount="{:.4f}".format((float(amount)/float(last_price))), price=last_price, side=side, currency_pair=f'{base}_{quote}')
```

Comment:
The idea of reducing this to 4 digits is basically good but it will probably not achieve the goal I assume. The fee will be taken from that amount which in most cases is around 0.015% and then we will end up with an amount with more than 4 digits anyway.
Also, why 4? Is there a specific requirement for using 4 digits?